.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+on: [push]
```

Comment:
Should we only do this on pushes to `main`? I think you had it this way and removed it for testing, so apologies if it's redundant.

```suggestion
on:
  push:
    branches:
      - main
```

Reply:
Right you are! Sorry, I guess I forgot to revert back my last test on this branch!